src/components/commons/HeroBanner.tsx

```diff
@@ -0,0 +1,103 @@
+import classNames from "classnames";
+import { SectionTitle } from "@components/commons/SectionTitle";
+import { Button } from "@components/commons/Buttons";
+import { StartExploring } from "@components/commons/StartExploring";
+
+// The HeroBanner component takes in title, subtitle, desc and HeroBannerBG
+// currently only COIN_ARROW, DFI_COIN, DIAMOND_COIN and CUBE image are available
+// it also takes in optional params on the customBgPosStyle
+// if cta button is required, to pass in hasCtaButton, ctaButtonText and ctaButtonLink
+// if start exploring button is required, to pass in hasStartExploringButton and startExploringJumpLink
+export function HeroBanner({
+  title,
+  subtitle,
+  desc,
+  customBgPosStyle,
+  heroBG,
+  hasCtaButton = false,
+  ctaButtonLink,
+  ctaButtonText,
+  hasStartExploringButton = false,
+  startExploringJumpLink,
+}: {
+  title: string;
+  subtitle: string;
+  desc: string;
+  customBgPosStyle?: string;
+  heroBG: HeroBannerBG;
+  hasCtaButton?: boolean;
+  ctaButtonLink?: string;
+  ctaButtonText?: string;
+  hasStartExploringButton?: boolean;
+  startExploringJumpLink?: string;
+}) {
+  return (
+    <div className="relative z-0">
+      <Container className={classNames("md:mt-[25px] mt-6")}>
+        <div
+          className={classNames(
+            "flex flex-col md:gap-y-5 gap-y-2",
+            "md:w-1/2 w-auto",
+            "md:pt-[88px] pt-[380px]",
+            "lg:mb-[278px] md:mb-[148px] mb-24"
+          )}
+        >
+          <SectionTitle text={title} />
+          <div
+            className={classNames(
+              "text-dark-1000 text-[40px] leading-[44px] tracking-[-0.02em]",
+              "md:text-5xl md:leading-[52px] md:tracking-normal",
+              "lg:text-[80px] lg:leading-[84px] md:tracking-normal"
+            )}
+          >
+            {subtitle}
+          </div>
+          <div
+            className={classNames(
+              "font-desc tracking-[0.03em] text-xl text-dark-700",
+              "lg:text-2xl tracking-normal"
+            )}
+          >
+            {desc}
+          </div>
+          {hasCtaButton && ctaButtonText && (
+            <Button
+              className="lg:py-4 py-3 md:w-1/2 w-auto lg:text-base text-sm md:mt-7 mt-10"
+              text={ctaButtonText}
+              href={ctaButtonLink}
+            />
+          )}
+        </div>
+        {hasStartExploringButton && startExploringJumpLink && (
+          <StartExploring startExploringJumpLink={startExploringJumpLink} />
+        )}
+      </Container>
+      <div
+        className={classNames(
+          bgMapping[heroBG],
+          "absolute bg-contain bg-no-repeat bg-right-top inset-0 z-[-1]",
+          "lg:h-[686px] md:h-[512px] h-[358px]",
+          customBgPosStyle ??
+            "xl:right-0 2xl:right-0 md:-right-[5%] -right-[3rem] md:-top-[5rem] top-0"
+        )}
+      />
+    </div>
+  );
+}
+
+export enum HeroBannerBG {
+  COIN_ARROW,
```

Comment:
why do we want to have enum and not just just accept a string prop?
I am thinking what if we'll need hero banner in more pages/components in future, this enum list will just keep growing? and you'll need to think of the enum naming

Reply:
meaning to take in the background assets url instead of using enum to handle ? 

i was thinking maybe it would be easier to handle on the pages side to just pass in the enum instead of the full string url

Reply:
if the banner images for all hero components are defined, meaning you know the total number of images will be used or there will be repeated pages that uses the same bg image, then i would agree enum could be useful, but right now the design is not completed, i think they will just have more diff hero banner images in the future.